crates/kitsune_p2p/types/src/tx2/tx2_pool_promote.rs

```diff
@@ -318,6 +318,7 @@ impl AsConHnd for ConItem {
 
             if let Err(e) = logic().await {
                 let reason = format!("{:?}", e);
+                tracing::error!(?e, "Closing writer");
```

Comment:
is it correct that this is error level? if this prints during normal operation, holo will say there's a problem when there isn't